stable/appmesh-controller/ci/values.yaml

```diff
@@ -0,0 +1,7 @@
+# CI testing values for appmesh-controller.
+
+region: us-west-2
+image:
+  repository: stefanprodan/aws-app-mesh-controller-for-k8s
```

Comment:
This is temporary, we will replace this once I get the docker repository set up.